### DISCUSSION TITLE
Improve error message when attempting to view logs for an in-progress run

### DISCUSSION THREAD

**Discussion by allanlewis** [renovatebot/renovate#28599] [Suggest an Idea]
### Tell us more.

When viewing an in-progress run, the following error message is shown:

> No log is available for this job
> Something went wrong and the log is not available

As per https://github.com/renovatebot/renovate/discussions/27994#discussioncomment-8830421, perhaps the error message could be updated to explain that viewing logs for in-progress runs isn't possible?

**Comment by jamietanna** [comment]
Thanks for the suggestion! 

We now have a clearer note in the logs page:

> **No log is available for this job**
> Logs will be available once the job is finished
